nvim-oxi/src/lua/pushable.rs

```diff
@@ -32,7 +32,7 @@ impl ObjectExt for Object {
             },
 
             kObjectTypeString => {
-                let string = ManuallyDrop::into_inner(self.data.string);
+                let string = ManuallyDrop::take(&mut self.data.string);
```

Comment:
reason to use `ManuallyDrop::take` instead of `into_inner`?

Reply:
can't move out of a type that implements `Drop`

Reply:
in that you should `mem::forget` the Object so it is not dropped twice.

Reply:
fixed in e90baefcabf92f71420f996d2118402d03280179